pycozmo/protocol_declaration.py

```diff
@@ -20,6 +21,7 @@
 
 FRAME_ID = b"COZ\x03RE\x01"
 MIN_FRAME_SIZE = len(FRAME_ID) + 1 + 2 + 2 + 2
+MAX_FRAME_SIZE = 1420 - MIN_FRAME_SIZE
```

Comment:
A better name for this may be `MAX_FRAME_PAYLOAD_SIZE`. Currently `MIN_FRAME_SIZE` is the total size of a frame with no payload but `MAX_FRAME_SIZE` is the maximum payload size only (minus the header`.

Also, I noticed that the robot drops frames, larger than 14+1037=1051 bytes with the following message:
```
WiFi.Messages.ProcessMessage.TooBig: Received message too big! 97(1375) > 1037
```

It seems that this is the maximum payload size.